src/input.rs

```diff
@@ -350,3 +368,28 @@ fn utf16le() {
     assert!(!res.unwrap());
     assert!(buffer.is_empty());
 }
+
+#[test]
+fn utf16le_issue3367() {
+    let content = b"\xFF\xFE\x0A\x4E\x00\x4E\x0A\x4F\x00\x52";
```

Comment:
It would be better to have an actual newline here for validation, rather than just reading until the end of the line.

Reply:
done :)